openproblems/tasks/chromatin_potential/datasets/scicar_chrom.py

```diff
@@ -0,0 +1,7 @@
+from ....tools.decorators import dataset
+from ....data.scicar_chrom import load_scicar_mouse_kidney
```

Comment:
Why not `from ....data.multimodal.scicar import load_scicar_mouse_kidney`?

Reply:
There is missing peak coordinate information, which is needed to model the TSS to peak distance.

Reply:
You should just make those obs available in the original dataset loader